tasks/kubevirt-disk-uploader/0.5.0/kubevirt-disk-uploader.yaml

```diff
@@ -16,7 +16,7 @@ spec:
     Automates the extraction of disk and uploads it to a container registry,
     to be used in multiple Kubernetes clusters.
   params:
-  - name: VM_NAME
+  - name: POD_NAME
```

Comment:
Another thing is, that you will have to define another env variable,since the pod can run in different namespace than VM

Reply:
Why will you have to define another env var?